Adapters/Vungle/ISVungleAdapter/build.gradle

```diff
@@ -51,7 +51,7 @@ android {
 
  dependencies {
     implementation 'com.ironsource.sdk:mediationsdk:7.2.7'
-    implementation 'com.github.vungle:vng-android-sdk:e1c598ff18'
+    implementation 'com.github.vungle:vng-android-sdk:7.0.0-early1-RC5'
```

Comment:
I have a question about this. Should this be the commit hash short or the tag? Does it matter to gradle?

Reply:
Both are fine, we can use commit hash, tag, and branch.